Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -123,7 +123,7 @@
   $SEC101/038.PostgreSqlCredentialsAdoSecret=(?i)(?:password|pwd)\s*=\s*(?P<secret>[^,;"'<\s\n]{8,128})(?:[,;"'<\s\n]|$)
   $SEC101/038.PostgreSqlCredentialsAdoResource=(?i)(?:database|db|dbname)\s*=\s*(?P<resource>[^,;"'<\s\n]+)(?:[,;"'<\s\n]|$)
 
-  $SEC101/041.RabbitMqCredentials=(?i)amqps?:\/\/(?P<id>[^:]+):(?P<secret>[^@]+)@(?P<host>[\w_\-\:]+)\/(?P<resource>[\w]+)(?:[^0-9a-z]|$)
+  $SEC101/041.RabbitMqCredentials=(?i)amqps?:\/\/(?P<id>[^:]+):(?P<secret>[^@\s\n]+)@(?P<host>[\w_\-\:]+)\/(?P<resource>[\w]+)(?:[^0-9a-z]|$)
```

Comment:
>\n [](http://example.com/codeflow?start=79&length=2)

\n is already implied by \s isn't it?

i think that's the case. if it *weren't*. i think your change would be incomplete without adding \r as well. 

But i think \s is sufficient

Reply:
you are write. \s already includes \n.

Reply:
I will also create a new new pr refactoring the rules that contains \n and \s.